src/command_line.py

```diff
@@ -4,8 +4,9 @@
 from str2bool import str2bool
 
 from src import __version__ as version
+from src import DAL
 from src.packagers import packager_type_codes, default_packager_type_code
-from src.providers import providers
+from src.providers import get_providers
 
 
 __parser = argparse.ArgumentParser(formatter_class=argparse.RawTextHelpFormatter)
```

Comment:
we need to upsert README.md

Reply:
Huh?

Reply:
Github code line marking is a bit buggy, what i meant is that we need to uldate our readme, you have introduced a whole new cli arguments. We must keep track with how one can use that tool

Reply:
When We'll write the configuration loader (which loads .yaml files, .json files, .env files, env variables and CLI parameters) we'll incorporate into that examples and better explenations

Reply:
Maybe even generate a '.md' file

Reply:
Even a small, simple README is fine for now

Reply:
I think that is out of scope and makes the `readme` dependant of the code itself, but as you wish